erpnext/accounts/report/customer_ledger_summary/customer_ledger_summary.py

```diff
@@ -242,6 +243,10 @@ def get_columns(self):
 				}
 			]
 
+		if self.filters.show_dr_cr:
+			columns.append(
+				{"label": _("Dr or Cr"), "fieldname": "dr_or_cr", "fieldtype": "Data", "width": 100}
```

Comment:
'Dr / Cr' would be more suitable. The literal word 'or' is mostly not used in this context.